scripts/MakeApiBaselines.ps1

```diff
@@ -4,6 +4,12 @@
     Creates API baseline files representing the current API surface exposed by this repo.
 #>
 
+if ($PSVersionTable.PSVersion.Major -lt 6) {
```

Comment:
Required for the `-LeafBase` argument. Even though I had a recent windows install, my PS version was 5.x